sdk/nodejs/provider.ts

```diff
@@ -39,7 +39,7 @@ export interface ProviderArgs {
      */
     readonly kubeconfig?: pulumi.Input<string>;
     /**
-     * If present, the namespace scope to use.
+     * If present, the default namespace to use. This flag is ignored for cluster-scoped resources.
```

Comment:
Can we add a note about how the namespace is used here? e.g. resources with their own namespace will not be overridden, unless they don't have one set - at which point we'll use the default.